db_master_deploy/deploy.sh

```diff
@@ -0,0 +1,379 @@
+#!/bin/bash
+MY_DIR=$(dirname $(readlink -f $0))
+source "$MY_DIR/includes.sh"
+
+display_usage() {
+    echo -e "Usage:\n$0 -s source_objects -t target_staging -a timestamp for BOD snapshot/archive (YYYYMMDD)"
+    echo -e "\t-s comma delimited list of source databases and/or tables - mandatory"
+    echo -e "\t-t target staging - mandatory choose one of '$targets'"
+    echo -e "\t-a is optional and only valid for BOD, if you dont enter a target the script will just create an archive/snapshot copy of the bod\n"
+}
+
+while getopts ":s:t:a:" options; do
+    case "$options" in
+        s)
+            source_objects=$OPTARG
+            IFS=',' read -ra array_source <<< "$source_objects"
+            ;;
+        t)
+            target=$OPTARG
+            ;;
+        a)
+            timestamp=$OPTARG
+            ;;
+        \? )
+            display_usage 1>&5 2>&6
```

Comment:
yes 5 and 6 are  used to bypass the default redirection of 1 and 2.
1 and 2 are prefixed with INFO: and ERROR: and are sent to the screen and to syslog.
https://github.com/geoadmin/deploy/blob/dev_ltclm_db_master_management_scripts/db_master_deploy/includes.sh#L45

everything sent to 5 and 6 is not prefixed and is not sent to syslog.